src/screens/TopicPageScreen/index.js

```diff
@@ -1,56 +1,35 @@
+import {useNavigation, useRoute} from '@react-navigation/native';
 import {useSafeAreaInsets} from 'react-native-safe-area-context';
+
 import BlockComponent from '../../components/BlockComponent';
 import ButtonAddPostTopic from '../../components/Button/ButtonAddPostTopic';
+import MemoizedListComponent from './MemoizedListComponent';
+import NavHeader from './elements/NavHeader';
+import ShareUtils from '../../utils/share';
 import TiktokScroll from '../../components/TiktokScroll';
-import {Context} from '../../context';
-import {setFeedByIndex, setTopicFeedByIndex, setTopicFeeds} from '../../context/actions/feeds';
+import TopicPageStorage from '../../utils/storage/custom/topicPageStorage';
+import dimen from '../../utils/dimen';
+import removePrefixTopic from '../../utils/topics/removePrefixTopic';
+import useCoreFeed from '../FeedScreen/hooks/useCoreFeed';
+import useFeedPreloadHook from '../FeedScreen/hooks/useFeedPreloadHook';
+import useViewPostTimeHook from '../FeedScreen/hooks/useViewPostTimeHook';
 import useOnBottomNavigationTabPressHook, {
   LIST_VIEW_TYPE
 } from '../../hooks/navigation/useOnBottomNavigationTabPressHook';
+import {COLORS} from '../../utils/theme';
+import {Context} from '../../context';
+import {downVote, upVote} from '../../service/vote';
 import {getFeedDetail} from '../../service/post';
 import {getTopicPages} from '../../service/topicPages';
 import {getTopics, getUserTopic} from '../../service/topics';
-import {downVote, upVote} from '../../service/vote';
-import dimen from '../../utils/dimen';
-import {normalize, normalizeFontSizeByWidth} from '../../utils/fonts';
-import {COLORS} from '../../utils/theme';
-import {linkContextScreenParamBuilder} from '../../utils/navigation/paramBuilder';
-import ShareUtils from '../../utils/share';
-import TopicPageStorage from '../../utils/storage/custom/topicPageStorage';
-import removePrefixTopic from '../../utils/topics/removePrefixTopic';
 import {getUserId} from '../../utils/users';
-import useCoreFeed from '../FeedScreen/hooks/useCoreFeed';
-import useFeedPreloadHook from '../FeedScreen/hooks/useFeedPreloadHook';
-import useViewPostTimeHook from '../FeedScreen/hooks/useViewPostTimeHook';
-import MemoizedListComponent from './MemoizedListComponent';
-import NavHeader from './elements/NavHeader';
-
-const styles = StyleSheet.create({
-  parentContainer: {
-    flex: 1
-  },
-  header: {
-    backgroundColor: COLORS.signed_primary,
-    justifyContent: 'center',
-    alignItems: 'center',
-    paddingTop: 40,
-    position: 'absolute',
-    top: 0,
-    left: 0,
-    right: 0,
-    zIndex: 2
-  },
-  headerText: {
-    fontSize: 20,
-    fontWeight: 'bold'
-  }
-});
+import {linkContextScreenParamBuilder} from '../../utils/navigation/paramBuilder';
+import {normalize, normalizeFontSizeByWidth} from '../../utils/fonts';
+import {setFeedByIndex, setTopicFeedByIndex, setTopicFeeds} from '../../context/actions/feeds';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import statements have been reordered. While this doesn't affect the functionality of the code, it's a good practice to maintain a consistent order for imports. This helps in readability and maintainability of the code. However, since there is no standard rule for ordering imports in JavaScript, this change is acceptable as long as it follows the project's conventions.

```javascript
import * as React from 'react';
import _ from 'lodash';
import SimpleToast from 'react-native-simple-toast';
import {Animated, Platform, StyleSheet, View} from 'react-native';
import {useNavigation, useRoute} from '@react-navigation/native';
import {useSafeAreaInsets} from 'react-native-safe-area-context';
import BlockComponent from '../../components/BlockComponent';
import ButtonAddPostTopic from '../../components/Button/ButtonAddPostTopic';
import MemoizedListComponent from './MemoizedListComponent';
import NavHeader from './elements/NavHeader';
import ShareUtils from '../../utils/share';
import TiktokScroll from '../../components/TiktokScroll';
import TopicPageStorage from '../../utils/storage/custom/topicPageStorage';
import dimen from '../../utils/dimen';
import removePrefixTopic from '../../utils/topics/removePrefixTopic';
import useCoreFeed from '../FeedScreen/hooks/useCoreFeed';
import useFeedPreloadHook from '../FeedScreen/hooks/useFeedPreloadHook';
import useViewPostTimeHook from '../FeedScreen/hooks/useViewPostTimeHook';
import useOnBottomNavigationTabPressHook, {
  LIST_VIEW_TYPE
} from '../../hooks/navigation/useOnBottomNavigationTabPressHook';
import {COLORS} from '../../utils/theme';
import {Context} from '../../context';
import {downVote, upVote} from '../../service/vote';
import {getFeedDetail} from '../../service/post';
import {getTopicPages} from '../../service/topicPages';
import {getTopics, getUserTopic} from '../../service/topics';
import {getUserId} from '../../utils/users';
import {linkContextScreenParamBuilder} from '../../utils/navigation/paramBuilder';
import {normalize, normalizeFontSizeByWidth} from '../../utils/fonts';
import {setFeedByIndex, setTopicFeedByIndex, setTopicFeeds} from '../../context/actions/feeds';
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->